clients/extension/styles/lighthouse.css

```diff
@@ -43,168 +52,122 @@ html, body {
 }
 
 .header__icon {
-  margin-right: 16px;
+  margin: 2px auto 0;
 }
 
-.header-titles {
-  display: flex;
-  flex-direction: column;
-  align-self: center;
-  flex: 1;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.errormsg {
+  color: var(--color-blue);
+  font-weight: 500;
+  text-align: center;
+}
+.errormsg:empty {
+  display: none;
 }
 
-.header-titles__main {
-  margin: 0;
-  font-family: 'Roboto', Arial, sans-serif;
-  color: currentColor;
-  font-weight: 300;
-  margin-bottom: 4px;
+.button:disabled {
+  background-color: #cccccc;
+  border: none;
+  color: #666666;
+  cursor: not-allowed;
 }
 
-.header-titles__url {
-  margin: 0;
-  font-size: 14px;
-  opacity: 0.74;
-  color: currentColor;
+.button--generate {
```

Comment:
can we get a hover style on this button? it feels lifeless.

Reply:
it's aliveeeeee